### ISSUE TITLE
Tests broken: panic on template expansion

### ISSUE THREAD

**Issue by geek1011** [just-install/registry#33]
````
$ go test -timeout 1h -v
=== RUN   TestRegistryReachableLinks
--- FAIL: TestRegistryReachableLinks (7.90s)
panic: template: expand:1: unexpected "}" in operand [recovered]
	panic: template: expand:1: unexpected "}" in operand
goroutine 7 [running]:
panic(0x71aea0, 0xc4204225f0)
	/home/travis/.gimme/versions/go1.7.4.linux.amd64/src/runtime/panic.go:500 +0x1a1
testing.tRunner.func1(0xc420080180)
	/home/travis/.gimme/versions/go1.7.4.linux.amd64/src/testing/testing.go:579 +0x25d
panic(0x71aea0, 0xc4204225f0)
	/home/travis/.gimme/versions/go1.7.4.linux.amd64/src/runtime/panic.go:458 +0x243
text/template.Must(0x0, 0x8e6ee0, 0xc4204225f0, 0x0)
	/home/travis/.gimme/versions/go1.7.4.linux.amd64/src/text/template/helper.go:23 +0x66
github.com/just-install/just-install.expandString(0xc420012dc0, 0x3e, 0xc420337f20, 0xc420197d10, 0xc420337f20)
	/home/travis/gopath/src/github.com/just-install/just-install/util.go:37 +0x1dc
github.com/just-install/just-install.(*RegistryEntry).ExpandString(0xc420363b30, 0xc420012dc0, 0x3e, 0xc420363b30, 0x50)
	/home/travis/gopath/src/github.com/just-install/just-install/registry.go:251 +0xcc
_/home/travis/build/just-install/registry.TestRegistryReachableLinks.func3(0xc420017bd0, 0xa, 0xc420363b30, 0x778fae, 0x3, 0xc420012dc0, 0x3e)
	/home/travis/build/just-install/registry/link_reachability_test.go:94 +0x7e
_/home/travis/build/just-install/registry.TestRegistryReachableLinks(0xc420080180)
	/home/travis/build/just-install/registry/link_reachability_test.go:104 +0x27f
testing.tRunner(0xc420080180, 0x7a5b28)
	/home/travis/.gimme/versions/go1.7.4.linux.amd64/src/testing/testing.go:610 +0x81
created by testing.(*T).Run
	/home/travis/.gimme/versions/go1.7.4.linux.amd64/src/testing/testing.go:646 +0x2ec
exit status 2
FAIL	_/home/travis/build/just-install/registry	7.908s
````

https://github.com/just-install/just-install/blob/master/util.go#L27-L40

**Comment by geek1011** [comment]
cc @lvillani @guiweber

**Comment by geek1011** [comment]
I think there is a extra bracket in one of the urls.

**Comment by guiweber** [comment]
If that's the issue, it would be great if the tester told us which URL is problematic. The document itself contains valid JSON, so it shouldn't be an issue.

**Comment by lvillani** [comment]
The testing code is due for a rewrite and a merge into the main executable (so that we can run the tests with a released version of just-install, on Windows). Btw, this particular issue should be fixed in 42551fb7ea896c94b9d7dc63c66d1ca70aad3a2f.

**Comment by geek1011** [comment]
@lvillani I can work on a rewrite of the testing code if I have time.
